src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -72,6 +75,15 @@ public UniquePersonList(UniquePersonList source) {
         internalList.addAll(source.internalList);
     }
 
+
+    /**
+     * Sort the list by alphabetical
+     */
+    public void sortByAlphabetical() {
+        internalList.sort((person1, person2) -> person1.getName().toString().compareToIgnoreCase(person2.getName().toString()));
```

Comment:
This line is a tad bit long. You can consider splitting this into two lines for improved readability.